persistence/provider/boltdb/eventstore.go

```diff
@@ -24,6 +23,13 @@ var (
 	// buffers.
 	eventStoreItemsBucketKey = []byte("items")
 
+	// eventStoreMessageIDsBucketKey is the key for a child bucket that indexes
+	// event message id against the event offsets.
+	//
+	// The keys are the string message IDs converted to bytes. The
+	// values are the event offsets encoded as 8-byte big-endian packets.
+	eventStoreMessageIDsBucketKey = []byte("message_ids")
```

Comment:
```suggestion
	eventStoreOffsetsBucketKey = []byte("offsets")
```

I made a similar suggestion on the memory implementation - we typically name the buckets for what they contain, now how they're keyed, if that makes sense.